### DISCUSSION TITLE
could not lock config file

### DISCUSSION THREAD

**Discussion by imap30** [Chia-Network/chia-blockchain#8636] [Support]
I'm running into this issue installing on my ubuntu server 20 with a gui installed:

error: could not lock config file /home/andrew/chia-blockchain/.git/modules/chia-blockchain-gui/config: Permission denied
fatal: could not set 'core.worktree' to '../../../chia-blockchain-gui'
Is there a fix?

**Comment by BrandtH22** [comment]
@imap30 , the newest version of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
